git_hg_sync/config.py

```diff
@@ -37,7 +37,7 @@ class SentryConfig(pydantic.BaseModel):
 
 class Config(pydantic.BaseModel):
     pulse: PulseConfig
-    sentry: SentryConfig | None = None
+    sentry: SentryConfig = SentryConfig(sentry_url="")
```

Comment:
Should this be SENTRY_DSN? Not sure if it's meant to be just a URL prefix, or to include the full DSN. If the latter, it should be renamed to `sentry_dsn` to align with current conventions.